test/check-machines-nics

```diff
@@ -140,26 +144,53 @@ class TestMachinesNICs(machineslib.VirtualMachinesCase):
-        b.assert_pixels("#vm-subVmTest1-networks",
-                        "vm-details-nics-card-edit-disabled",
-                        ignore=["#vm-subVmTest1-network-1-mac",
-                                "#vm-subVmTest1-network-1-ipv4-address",
-                                "#vm-subVmTest1-network-1-source",
-                                "#vm-subVmTest1-network-2-mac",
-                                "#vm-subVmTest1-network-2-ipv4-address",
-                                "#vm-subVmTest1-network-2-source",
-                                "#vm-subVmTest1-network-3-mac",
-                                "#vm-subVmTest1-network-3-ipv4-address",
-                                "#vm-subVmTest1-network-3-source"],
-                        # MAC and IP address values are not static, and their values affect the width of
-                        # their columns and neighbouring columns
-                        # With medium layout this variable width of the columns makes pixel tests references of
-                        # the table different each test run
-                        skip_layouts=["medium", "rtl"])
+        b.assert_pixels(
+            "#vm-subVmTest1-networks",
+            "vm-details-nics-card-edit-disabled",
+            ignore=[
+                "#vm-subVmTest1-network-1-mac",
+                "#vm-subVmTest1-network-1-ipv4-address-0",
+                "#vm-subVmTest1-network-1-source",
+                "#vm-subVmTest1-network-2-mac",
+                "#vm-subVmTest1-network-2-ipv4-address-0",
+                "#vm-subVmTest1-network-2-source",
+                "#vm-subVmTest1-network-3-mac",
+                "#vm-subVmTest1-network-3-ipv4-address-0",
+                "#vm-subVmTest1-network-3-source",
+            ],
+            # MAC and IP address values are not static, and their values affect the width of
+            # their columns and neighbouring columns
+            # With medium layout this variable width of the columns makes pixel tests references of
+            # the table different each test run
+            skip_layouts=["medium", "rtl"],
+        )
 
         # Start vm and wait until kernel is booted
         testlib.wait(lambda: "1" in self.machine.execute("virsh domifaddr subVmTest1 | grep 192.168.122. | wc -l"),
                      delay=3)
+
+        # To add more IPs we need to know which interface to add them to.
+        # This differs depending on local and cloud testing
+        iface = "$(ip -br address show | grep '192.168.122.' | cut -d ' ' -f 1)"
+        # Add a secondary IP which should show
+        secondary_ip = "192.168.123.100/24"
+        self.machine.execute(
+            "virsh qemu-agent-command subVmTest1 "
+            '\'{"execute": "guest-exec", "arguments":'
+            '{ "path": "ip", "arg": [ "address","add","' + secondary_ip + '", "dev", "' + iface + '" ], '
+            '"capture-output": true }}\''
+        )
```

Comment:
Can't run tests locally without rebooting atm so won't continue today. But this did work locally but doesn't seem to work on the host. Had a different solution before that didn't work either, and now with this cleaner approach I also struggle to get it to add the IP.

```bash
cockpit-machines on  nics/capture-all-ips [!] via  v22.18.0 via 🐍 v3.13.7 
❯ ip -br address show | grep '10.35.30.' | cut -d ' ' -f 1
enp5s0
```

I suspect that the interface is correct but not sure why this doesn't work whatsoever. Gonna give it another try by waiting for the IP to appear